Makefile

```diff
@@ -28,7 +28,7 @@ cover:
 	find cover_db -type d -exec chmod 755 {} \;
 
 tags:
-	/usr/bin/ctags -R --languages=perl elevate-cpanel t
+	/usr/bin/ctags -R --languages=perl --extra=+q script lib t
```

Comment:
what specifically is broken here or being fixed?

Reply:
What's being fixed here is my (and others') ability to use Ctrl-] / Ctrl-T in vim to jump to definitions. It's totally unrelated but also absolutely immaterial to the user-facing functionality or the build/release process. If you really want a separate PR for this...